### PR TITLE
Fix observer value + add administrator table to merge

### PR DESCRIPTION
**Pull request by SebastienPeillet**
This PR :
- fixes observer value during validation
- add administration table to merge
- pseudo validation before merge to transfer value from administration table to environment/sighting/followers table

### CODE REVIEW

**Comment by SebastienPeillet** [comment]
Works on Windows too ! @pblottiere any comment or I merge ?

File: src/core/gps.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def _toDoInsideLoop(self):
             if self._gps:
                 self._gps.close()
                 self._gps = None
-            if not os.environ.get("SAMMO_DEBUG"):
+            if os.environ.get("SAMMO_DEBUG"):
                 print("GPS offline - exception when trying to read")
```
Comment (pblottiere) [inline src/core/gps.py:61]: We could probably use the logger instead of `print` in this case and this way hide the `SAMMO_DEBUG` mechanism?

**Comment by pblottiere** [comment]
> Works on Windows too ! @pblottiere any comment or I merge ?

Nice :tada:! I just have a small comment but it's not related to the core of this PR. So +1 to merge and we'll take a look later to the print/logger.

File: src/core/gps.py (hunk 1 above)
Comment (SebastienPeillet) [reply src/core/gps.py:61]: Yes, it's true I even forgot we had a Logger class in sammo :sweat_smile:

**Pull request merged**